### PR TITLE
make glyph ordering conventions align with create_glyph_type_varibles function

### LINKED ISSUE #13: add conventions for missing segments
and specify where the others ones belong in the application order, including:

  "͊",  # denasalized (combining not tilde above)
  "͋",  # nasal emission (combining homothetic)
  "̬",  # stiff (combining caron below)
  "͇",  # non-sibilant (combining equals sign below)
  "͈",  # fortis (combining double vertical line below)
  "͉",  # lenis (combining left angle below)
  "̆",  # short (combining breve)

### PR DESCRIPTION
**Pull request by drammock**
closes #13

### CODE REVIEW

File: conventions.md [hunk 1]
```diff
@@ -95,14 +95,15 @@ Each representation of a phoneme is first normalized into a canonical decomposit
...
+1. **Place features:** velarized/pharyngealized ~~`̴`~~, linguolabial ~~`̼`~~, dental ~~`̪`~~, apical ~~`̺`~~, laminal ~~`̻`~~, advanced ~~`̟`~~, retracted ~~`̠`~~
+2. **Manner features:** non-sibilant ~~`͇`~~, raised ~~`̝`~~, lowered ~~`̞`~~, advanced tongue root ~~`̘`~~, retracted tongue root ~~`̙`~~, frictionalized ~~`͓`~~
+3. **Secondary articulations:** more round ~~`̹`~~, less round ~~`̜`~~
+4. **Laryngeal settings:** creaky ~~`̰`~~, breathy ~~`̤`~~, voiced ~~`̬`~~, stiff ~~`̬`~~, devoiced (below) ~~`̥`~~, devoiced (above) ~~`̊`~~, fortis ~~`͈`~~, lenis ~~`͉`~~
+5. **Length:** short ~~`̆`~~
+6. **Syllabicity:** syllabic ~~`̩`~~ and non-syllabic markers ~~`̯`~~
+7. **Vowel quality modifications**: nasalized ~~`̃`~~, denasalized ~~`͊`~~, centralized ~~`̈`~~, mid-centralized ~~`̽`~~
+8. **Stop release:** unreleased `̚`
+9. **Spacing modifier letters:** rhotic hook `˞`, nasal release `ⁿ`, lateral release `ˡ`, labialized `ʷ`, palatalized `ʲ`, labial-palatalized `ᶣ`, velarized `ˠ`, pharyngealized `ˤ`, glottalized `ˀ`, schwa-like release `ᵊ`, epilaryngeal source `ᴱ`, aspirated `ʰ`, breathy aspirated `ʱ`, ejective `ʼ`, long `ː`, half long `ˑ`
```
Comment (bambooforest) [inline conventions.md:106]: Here we call it `rhotic wing`: https://github.com/phoible/dev/blob/master/scripts/aggregation-helper-functions.R#L362 , but I prefer what you have here (in Unicode it's official name is also hook, i.e. "modifier letter rhotic hook")

File: conventions.md [hunk 2]
```diff
@@ -95,14 +95,15 @@ Each representation of a phoneme is first normalized into a canonical decomposit
...
-4. **Laryngeal settings:** creaky ~~`̰`~~, breathy ~~`̤`~~, voiced ~~`̬`~~, devoiced (below) ~~`̥`~~, devoiced (above) ~~`̊`~~
-5. **Syllabicity:** syllabic ~~`̩`~~ and non-syllabic markers ~~`̯`~~
-6. **Vowel quality modifications**: nasalized ~~`̃`~~, centralized ~~`̈`~~, mid-centralized ~~`̽`~~
-7. **Stop release:** unreleased `̚`
-8. **Spacing modifier letters:** rhotic hook `˞`, nasal release `ⁿ`, lateral release `ˡ`, labialized `ʷ`, palatalized `ʲ`, velarized `ˠ`, pharyngealized `ˤ`, glottalized `ˀ`, aspirated `ʰ`, ejective `ʼ`, long `ː`, half long `ˑ`
+1. **Place features:** velarized/pharyngealized ~~`̴`~~, linguolabial ~~`̼`~~, dental ~~`̪`~~, apical ~~`̺`~~, laminal ~~`̻`~~, advanced ~~`̟`~~, retracted ~~`̠`~~
+2. **Manner features:** non-sibilant ~~`͇`~~, raised ~~`̝`~~, lowered ~~`̞`~~, advanced tongue root ~~`̘`~~, retracted tongue root ~~`̙`~~, frictionalized ~~`͓`~~
+3. **Secondary articulations:** more round ~~`̹`~~, less round ~~`̜`~~
+4. **Laryngeal settings:** creaky ~~`̰`~~, breathy ~~`̤`~~, voiced ~~`̬`~~, stiff ~~`̬`~~, devoiced (below) ~~`̥`~~, devoiced (above) ~~`̊`~~, fortis ~~`͈`~~, lenis ~~`͉`~~
```
Comment (bambooforest) [inline conventions.md:101]: @drammock - we seem to support both above and below devoice markers:

https://github.com/phoible/dev/blob/master/scripts/aggregation-helper-functions.R#L400-L401

We discuss the issue here: #225 -- and I suppose I'm the bottle neck on going in and fixes the sources and doing a PR. I'll bump this up on my list.

File: conventions.md (hunk 1 above)
Comment (drammock) [reply conventions.md:106]: yeah, I prefer "hook" too. I'll try to remember to change "wing" next time I'm editing that file.

File: conventions.md (hunk 2 above)
Comment (drammock) [reply conventions.md:101]: since we already have a separate issue to track the above/below devoicing ring, I'll consider it not a blocker for this PR.

**Pull request merged**